cognite_toolkit/cdf_tk/load/_resource_loaders.py

```diff
@@ -1375,14 +1375,29 @@ def load_resource(
             print(f"  [bold green]INFO:[/] File pattern detected in {filepath.name}, expanding to all files in folder.")
             file_data = files_metadata.data[0]
             ext_id_pattern = file_data.external_id
+            filename_pattern = file_data.name
```

Comment:
This name confused me, suggest just calling it
 ```suggestion
            filename = file_data.name
```